easybuild/easyblocks/b/bazel.py

```diff
@@ -112,13 +124,43 @@ def configure_step(self):
         # See https://github.com/bazelbuild/bazel/issues/10377
         bazel_args += ' --host_javabase=@local_jdk//:jdk'
 
+        # Link C++ libs statically, see https://github.com/bazelbuild/bazel/issues/4137
+        env.setvar('BAZEL_LINKOPTS', '-static-libstdc++:-static-libgcc')
+        env.setvar('BAZEL_LINKLIBS', '-l%:libstdc++.a')
+
         env.setvar('EXTRA_BAZEL_ARGS', bazel_args)
+        env.setvar('EMBED_LABEL', self.version)
+        env.setvar('VERBOSE', 'yes')
 
     def build_step(self):
         """Custom build procedure for Bazel."""
-        cmd = '%s ./compile.sh' % self.cfg['prebuildopts']
+        # The initial bootstrap of bazel is done in TMPDIR
+        cmd = 'TMPDIR="%s" %s bash -c "set -x && ./compile.sh"' % (self.bazel_tmp_dir, self.cfg['prebuildopts'])
```

Comment:
I think that it would be simpler and more readable to first execute `env.setvar('TMPDIR', self.bazel_tmp_dir) and then set `cmd` as it is already done. Is there any downside to this approach?

Reply:
Yes: it overwrites TMPDIR for all subsequent executions which I don't want. I want to limit it to this single execution and make that visible in the log. However I can make it a list, wait...

Reply:
Better?: https://github.com/easybuilders/easybuild-easyblocks/pull/2285/commits/f4f1ae72d37b0fdfbf254129a2530e88df067b4f

Reply:
Yes, much better.